lib/gui/app.js

```diff
@@ -79,17 +79,24 @@ app.run(() => {
     '\\____/ \\__\\___|_| |_|\\___|_|',
     '',
     'Interested in joining the Etcher team?',
-    'Drop us a line at join+etcher@resin.io'
+    'Drop us a line at join+etcher@resin.io',
+    '',
+    `Version = ${packageJSON.version}, Type = ${packageJSON.packageType}`
```

Comment:
But then again if we include the OS, do we also include the CPU architecture, etc. ? So rather than diving down the rabbit hole perhaps just the `Version` and `Type` are sufficient for now.